apps/condo/domains/notification/constants.js

```diff
@@ -29,6 +30,13 @@ const MESSAGE_META = {
         userPhone: { defaultValue: '', required: false },
         organizationName: { defaultValue: 'ORGANIZATION', required: false },
     },
+    [SHARE_TICKET_MESSAGE_TYPE]: {
```

Comment:
What about address + unit name

Reply:
if I understood correctly, they are not represented in the layout
![image](https://user-images.githubusercontent.com/1762825/127348164-9bcd0338-8b4b-41a1-a439-46aa3bd45577.png)